frontend/frontendAngular/src/app/components/navbar/navbar.component.ts

```diff
@@ -1,37 +1,28 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
 import { LocalStorageLoginService } from '../../services/local-storage-login.service';
-import { MessageService } from '../../services/message-service.service';
 
 @Component({
   selector: 'app-navbar',
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.scss']
 })
 export class NavbarComponent implements OnInit {
-  public url1!: string;
-  name = 'Get Current Url Route Demo';
-  currentRoute!: string;
 
   constructor(
     public router: Router,
-    private messageService: MessageService,
     public localStorage: LocalStorageLoginService
   ) { }
 
   ngOnInit(): void {
   }
 
-  showUser() {
-    this.messageService.showSuccess("Registrado com sucesso", "Ok");
-  }
-
   direct() {
-    if (this.localStorage.get("isCompany")) {
```

Comment:
substituir por if (this.localStorage.decodePayloadJWT(this.localStorage.get("token")).cnpj != null) {